build.sbt

```diff
@@ -79,21 +79,21 @@ scalaVersion := "2.12.0"
 
 libraryDependencies ++= Seq(
   ws,
-  "uk.gov.hmrc"               %% "bootstrap-play-26"              % "1.3.0",
-  "uk.gov.hmrc"               %% "domain"                         % "5.6.0-play-26",  //see if can be removed after upgrade
-  "uk.gov.hmrc"               %% "govuk-template"                 % "5.48.0-play-26",
-  "uk.gov.hmrc"               %% "play-ui"                        % "8.7.0-play-26",
-  "uk.gov.hmrc"               %% "play-partials"                  % "6.9.0-play-26",
+  "uk.gov.hmrc"               %% "bootstrap-play-26"              % "1.7.0",
+  "uk.gov.hmrc"               %% "domain"                         % "5.8.0-play-26",  //see if can be removed after upgrade
```

Comment:
Could we get rid of the two comments in this file?

Reply:
Removed these